.circleci/config.yml

```diff
@@ -392,11 +392,11 @@ workflows:
       - clear-model-instances:
           context: trade-tariff-fpo-models-development
 
-  bi-weekly-automated-model-build:
-    triggers:
-      - schedule:
-          cron: "0 0 1,15 * *"    # 1st and 15th of every month at midnight UTC
-          <<: *filter-main
-    jobs:
-      - automated-build-model:
-          context: trade-tariff-fpo-models-development
+  # bi-weekly-automated-model-build:
```

Comment:
Do we not need this job anymore? 👀

Reply:
I could probably do with having a look at it. Ideally we keep it and have the updated data sources reflected in the model in an unattended way. I might catch up with @amberstarlight on this separately

Reply:
Did you take a look at this and agree to remove it after all? I'm a little confused